tests/snapshot/snapshot_restore_test.go

```diff
@@ -86,7 +88,20 @@ func (s *SnapshotRestoreTestSuite) TestTfpSnapshotRestore() {
 			s.T().Skip("RKE1 is not supported")
 		}
 
-		if strings.Contains(s.terraformConfig.Module, "ec2") && (s.terraformConfig.ETCD.S3 != nil || s.terraformConfig.ETCDRKE1.BackupConfig.S3BackupConfig != nil) {
+		var rke1S3 *client.S3BackupConfig
```

Comment:
Since RKE1 isn't supported, why include this block here? We only need to have RKE2 or K3s.

Reply:
Fixed